packages/blocks/src/api/parser/index.js

```diff
@@ -53,6 +53,7 @@ import { applyBuiltInValidationFixes } from './apply-built-in-validation-fixes';
 /**
  * @typedef  {Object}  ParseOptions
  * @property {boolean} __unstableSkipMigrationLogs If a block is migrated from a deprecated version, skip logging the migration details.
+ * @property {boolean} __unstableAutoP             Whether to run autoP in freeform content.
```

Comment:
Two nit picks:
* I don't remember ever seeing `autop` cased `autoP`
* A "skip" flag seems a little more fitting, but I don't feel strongly about it

```suggestion
 * @property {boolean} __unstableSkipAutop         Whether to skip autop when processing freeform content.
```

Reply:
Good shout, I've made the suggested changes 👍